e2e/test/scenarios/custom-column/custom-column.cy.spec.js

```diff
@@ -799,7 +799,7 @@ describe("scenarios > question > custom column", () => {
           formula: 'notIn("Gadget", [Vendor], [Category])',
           name: "InColumn",
         });
-        cy.button("Update").click();
+        cy.button("Update").should("not.be.disabled").click();
```

Comment:
Cypress has [actionability checks](https://docs.cypress.io/app/core-concepts/interacting-with-elements#Actionability) before interacting with elements. This `.should("not.be.disabled")` assertion doesn’t do anything extra